pkg/terraform/util.go

```diff
@@ -87,3 +87,31 @@ func CreateTerraformBackendFile(assetDir, data string) error {
 
 	return nil
 }
+
+// PrepareTerraformRootDir creates a directory named path including all
+// required parents.
+// An error is returned if the directory already exists.
+func PrepareTerraformRootDir(path string) error {
```

Comment:
This function only wraps `os.MkdirAll`, so the only value it provides is to deduplicate error message. I think we could remove it instead.

Also, comment says, that if directory exists, it returns error. I don't think this is correct.